i18n/en.json

```diff
@@ -1432,7 +1432,7 @@
   },
   {
     "id": "api.scheme.patch_scheme.license.error",
-    "translation": ""
+    "translation": "License does not support update permissions schemes"
```

Comment:
`Your license does not support updating permissions schemes.`

Reply:
The file includes more examples of `License does not support... whatever` than `Your license does not support... whatever`, we can change it everywhere if you want.